src/drivers/telemetry/frsky_telemetry/frsky_telemetry.cpp

```diff
@@ -199,9 +216,6 @@ static void usage()
  */
 static int frsky_telemetry_thread_main(int argc, char *argv[])
 {
-	/* Default values for arguments */
-	const char *device_name = "/dev/ttyS6"; /* USART8 */
```

Comment:
This can lead to a memory access violation with the following steps:
```
frsky_telemetry start -d /dev/some/device
frsky_telemetry stop
frsky_telemetry start  <- accesses `device_name` which was freed via `frsky_telemetry stop`.
```
It can be fixed by initializing device_name here:
```
device_name = "/dev/ttyS6"; /* USART8 */
```
And initializing it with NULL above, and also to be sure, setting it to NULL when the main thread exits.

Reply:
WIP